### ISSUE TITLE
Before/After Scenario not working in Serenity BDD

### ISSUE THREAD

**Issue by jaspreetmanchinda** [serenity-bdd/serenity-jbehave#117]
I have already posted a question in other forum too.
http://stackoverflow.com/questions/42860315/before-after-scenario-not-working-in-jbehave-serenity-bdd

When I run the test the code does not enter the before and after scenario block.
The version i am using is 
<serenity.version>1.2.3-rc.5</serenity.
<serenity.jbehave.version>1.21.0</serenity.jbehave.version>

**Comment by wakaleo** [comment]
@BeforeScenario and @AfterScenario are JBehave annotations and are handled by JBehave - Serenity doesn't do anything special to run them, as they should be run by JBehave. Is there something special that you need to do with JBehave to use them? Can you create a sample JBehave project where they work correctly?

**Comment by jaspreetmanchinda** [comment]
Thanks for the quick response.
I will try with a sample jbehave project and share the update

**Comment by jaspreetmanchinda** [comment]
I created two sample project of below archetype
1.) serenity-jbehave-archetype version 1.2.0
2.)jbehave-simple-archetype version 4.1

and tried before and after scenario in both the sample projects its working fine
one thing that I noticed is when I run the default feature file  with before and after it executes, but when I execute it with my feature file it doesn't.
I am testing the rest service, Have three layers: one Step file GWT, one step file @steps which has the implementation and one common util which have the get, post implementation and other common implementation.

What can be the case where before and after wont execute?
